mkdocs_theme.yml

```diff
@@ -0,0 +1,33 @@
+site_name: OpenSAFELY documentation
```

Comment:
This file isn't required for the main docs, but at the moment it's the only way I found to get the docs to run from databuilder and pull in the parent config